src/components/modal/modal-add-edit-content.tsx

```diff
@@ -43,169 +60,158 @@ const ModalAddEditContent = ({
   const [releaseDate, setReleasedate] = useState(
     editableMode ? formData?.release_date : ``
   );
+  return (
+    <Formik
+      initialValues={{
+        genres,
+        title,
+        overview,
+        posterPath,
+        runtime,
+        releaseDate,
+      }}
+      validationSchema={validationSchema}
+      onSubmit={(values) => {
+        onModalSubmit({
+          ...formData,
+          genres: values.genres,
+          title,
+          overview,
+          poster_path: posterPath,
+          runtime,
+          release_date: releaseDate,
+        });
+      }}
+    >
+      {({ submitForm }) => (
+        <Form>
+          <>
+            <FormikTextField
+              label="Title"
+              name="title"
+              variant="filled"
+              color="secondary"
+              className={classes.formControl}
+              InputProps={{ className: classes.title }}
+              InputLabelProps={{
+                classes: {
+                  root: classes.label,
+                  focused: classes.focusedLabel,
+                },
+              }}
+              value={title}
```

Comment:
You also don't need `value` and `onChange` props, `formik` will control this values itself.
Also, please checkout `className`, `InputProps` and `InputLabelProps` props, it seems to me that they override each other and some of this props are excessive.